### PR TITLE
create new feature flag allow-status-page

### PR DESCRIPTION
**Pull request by lperry25**
create new feature flag allow-status-page to allow tenants to easily see status pages

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: flags/src/main/java/com/yahoo/vespa/flags/PermanentFlags.java [hunk 1]
```diff
@@ -367,6 +367,12 @@ public class PermanentFlags {
             "Takes effect on next redeployment",
             INSTANCE_ID);
 
+    public static final UnboundBooleanFlag ALLOW_STATUS_PAGE = defineFeatureFlag(
+            "allow-status-page", false,
+            "Shows link to status page for nodes of a specific tenant",
```
Comment (freva) [inline flags/src/main/java/com/yahoo/vespa/flags/PermanentFlags.java:372]: "status page" is a little vague, this should at least mention that this is the "cluster controller status page"

Comment (lperry25) [reply flags/src/main/java/com/yahoo/vespa/flags/PermanentFlags.java]: I updated the description and renamed the flag now

**Review by freva — DISMISSED** [review]

**Comment by kkraune** [comment]
This is not necessarily just for the cluster controller status pages, it is also for searchnode in another view

**Review by kkraune — CHANGES_REQUESTED** [review]
rename to more generic

**Comment by freva** [comment]
Reverted back to generic name, added TENANT_ID dimension. PTAL @kkraune

**Comment by kkraune** [comment]
looks good to me!

**Review by freva — APPROVED** [review]

**Review by aressem — APPROVED** [review]

**Review by kkraune — APPROVED** [review]

**Pull request merged**